src/receive_imf.rs

```diff
@@ -388,6 +388,7 @@ pub async fn from_field_to_contact_id(
     }
 }
 
+// Creates a `received message` from given parts
```

Comment:
It also inserts a Message into the `msgs` database table for every part in mime_parser.parts. It then returns the `msg_ids` of the inserted messages and some extra info in `ReceivedMsg`.

BTW, you should put three `///` at the start to mark a doc comment.